### PR TITLE
stat: change to full path in token

### PR DESCRIPTION
**Pull request by iridacea**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/stat.md [hunk 1]
```diff
@@ -5,24 +5,24 @@
 
 - Show file properties such as size, permissions, creation and access dates among others:
```
Comment (EmilySeville7cfg) [inline pages/common/stat.md:6]: The following edit can be applied later in another PR:

```suggestion
- Show a specific file properties such as size, permissions, creation and access dates among others:
```

All descriptions in all pages should be standardized.

File: pages/common/stat.md [hunk 2]
```diff
@@ -5,24 +5,24 @@
...
 
-`stat {{file}}`
+`stat {{path/to/file}}`
 
 - Same as above but in a more concise way:
 
-`stat -t {{file}}`
+`stat -t {{path/to/file}}`
 
 - Show filesystem information:
```
Comment (EmilySeville7cfg) [inline pages/common/stat.md:14]: The following edit can be applied later in another PR:

```suggestion
- Show a filesystem information:
```

File: pages/common/stat.md [hunk 3]
```diff
@@ -5,24 +5,24 @@
...
 
-`stat -f {{file}}`
+`stat -f {{path/to/file}}`
 
 - Show only octal file permissions:
 
-`stat -c "%a %n" {{file}}`
+`stat -c "%a %n" {{path/to/file}}`
 
 - Show owner and group of the file:
```
Comment (EmilySeville7cfg) [inline pages/common/stat.md:22]: The following edit can be applied later in another PR:

```suggestion
- Show an owner and a group of a specific file:
```

File: pages/common/stat.md [hunk 4]
```diff
@@ -5,24 +5,24 @@
...
 
-`stat -c "%U %G" {{file}}`
+`stat -c "%U %G" {{path/to/file}}`
 
 - Show the size of the file in bytes:
```
Comment (EmilySeville7cfg) [inline pages/common/stat.md:26]: The following edit can be applied later in another PR:


```suggestion
- Show a size of a specific file in bytes:
```

**Review by EmilySeville7cfg — APPROVED** [review]
No changes are requested because all suggestions can be applied in another PR by this PR author or by somebody else.

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! ❤️

File: pages/common/stat.md (hunk 1 above)
Comment (iridacea) [reply pages/common/stat.md:6]: This and three suggestions below are implemented  in #9438

**Pull request merged**